app/models/service_template.rb

```diff
@@ -371,7 +371,7 @@ def create_resource_actions(ae_endpoints)
   end
 
   def self.create_from_options(options)
-    create(options.except(:config_info).merge(:options => { :config_info => options[:config_info] }))
+    create(options.except(:config_info, :manager_id).merge(:options => { :manager_id => options[:manager_id], :config_info => options[:config_info] }))
```

Comment:
After talking to @syncrou I realized this is just going to lead to a `nil` if `:manager_id` is not provided.